docs/dev_guide/ceph_s3_tests/ceph_s3_tests_guide.md

```diff
@@ -88,21 +91,33 @@ kubectl apply -f src/test/system_tests/ceph_s3_tests/test_ceph_s3_job.yml
 kubectl logs job/noobaa-tests-s3 -f
 ```
 
```

Comment:
Since no longer all the tests are passing when running them on the local machine I would add a note about this:

```suggestion
Running all the tests on a local machine takes about 20 minutes. With the current setting, all tests should pass, but there are cases where the endpoint restarts and causes a test to fail. You can also run a single test to be sure that it passes.
```

For example, after the job run, I saw a test that was failing: `CONSOLE:: CEPH TEST FAILED TESTS SUMMARY: 1 failed tests
s3tests_boto3/functional/test_s3.py::test_multipart_copy_versioned`
so I checked the status of the pods: `kubectl get pods` and saw the endpoint had a restart.

BTW, I run this test twice and it causes the restarts of the endpoint...
In the second run it was with 2 restarts and 2 failing tests:
`CONSOLE:: CEPH TEST FAILED TESTS SUMMARY: 2 failed tests`
`s3tests_boto3/functional/test_s3.py::test_multipart_copy_versioned`
`s3tests_boto3/functional/test_s3select.py::test_json_column_sum_min_max`